### PR TITLE
56: [TDD] Identify even/odd numbers

### PR DESCRIPTION
**Pull request by Najlaalrajhia98**

### CODE REVIEW

**Review by AnwaarAlbusaidi — APPROVED** [review]
well Done

File: src/test/java/iotest/EvenOddTest.java [hunk 1]
```diff
@@ -22,10 +22,9 @@ void test() {
```
Comment (FatemaAlbalushi) [inline src/test/java/iotest/EvenOddTest.java]: Remove the unneeded space.

File: src/test/java/iotest/EvenOddTest.java [hunk 2]
```diff
@@ -22,10 +22,9 @@ void test() {
   
         
         // Tests the Odd num
```
Comment (FatemaAlbalushi) [inline src/test/java/iotest/EvenOddTest.java:21]: Try to test each case in spreat method.

**Review by FatemaAlbalushi — COMMENTED** [review]
Good job

**Review by Maryam313 — APPROVED** [review]
Great job Dear..!

**Review by FatemaAlbalushi — APPROVED** [review]

**Pull request merged**